packages/cli/src/commands/screenshot-diff/screenshot-diff.command.ts

```diff
@@ -158,7 +158,7 @@ export const checkScreenshotDiffResult = ({
   headReplayId: string;
   results: ScreenshotDiffResult[];
   diffOptions: ScreenshotDiffOptions;
-}): void => {
+}): ScreenshotDiffError | null => {
```

Comment:
nitpick: to be even more explicit, the return type could be something like `ScreenshotMatch | ScreenshotDiffError`, or `"match" | ScreenshotDiffError`.

Reply:
Yep I like that. I'll do something like ScreenshotsMatch | ScreenshotsDoNotMatch, and change ScreenshotDiffError to be a ScreenshotsDoNotMatch `{ type: "error", message: ... }` kind of thing instead of a literal Error (we no longer actually throw it anywhere).